src/components/ColumnNumber/index.js

```diff
@@ -0,0 +1,21 @@
+import { Cell } from 'components/Cell'
+import { Column } from 'components/Column'

```

Comment:
import {Cell, Column} from 'components'

Reply:
Refactored all folders / components / imports to follow this.